Index: RT5592_Linux_SoftAP_v2.7.x.x/rt5592/os/linux/rt_linux.c
===================================================================
--- RT5592_Linux_SoftAP_v2.7.x.x.orig/rt5592/os/linux/rt_linux.c
+++ RT5592_Linux_SoftAP_v2.7.x.x/rt5592/os/linux/rt_linux.c
@@ -921,8 +921,8 @@ static inline void __RtmpOSFSInfoChange(
 		pOSFSInfo->fsgid = current->fsgid;
 		current->fsuid = current->fsgid = 0;
 #else
-		pOSFSInfo->fsuid = current_fsuid();
-		pOSFSInfo->fsgid = current_fsgid();
+		pOSFSInfo->fsuid = 0;//current_fsuid();
+		pOSFSInfo->fsgid = 0;//current_fsgid();
 #endif
 		pOSFSInfo->fs = get_fs();
 		set_fs(KERNEL_DS);
